### ISSUE TITLE
Feature Request: Align Tasks UUID between gvmd / ospd-openvas

### ISSUE THREAD

**Issue by wisukind** [greenbone/ospd-openvas#287]
**Environment:**
```
Ubuntu 18.04
Greenbone Vulnerability Manager 9.0.1~git-e250176b-gvmd-9.0
GIT revision e250176b-gvmd-9.0
Manager DB revision 221
gvm@ov-master-eqi:~$ gsad --version
Greenbone Security Assistant 9.0.1~git-9fb2e63cd-gsa-9.0
gvm@ov-master-eqi:~$ openvas --version
OpenVAS 7.0.1
gvm-libs 11.0.1
```
**Current behaviour:**
Gvmd use one UUID for a task; while ospd-openvas use a different one for the same task.This make issues & bug tracking very difficult in a master / slave architecture when you have multiple scans running.

**Expected behaviour:**
Share the same tasks UUIDs between gvmd and ospd-openvas.

**Comment by bjoernricks** [comment]
I think #326 will address this issue.

**Comment by jjnicola** [comment]
Yes, already addressed with PR #326. This PR will not be backported. Therefore, closing the issue.
